lib/reducers/create-otp-reducer.js

```diff
@@ -542,11 +556,23 @@ function createOtpReducer (config, initialQuery) {
           }
         })
       case 'SET_ROUTER_ID':
-        const routerId = action.payload || 'default'
+        const routerId = action.payload
+        // Store original path value in _path variable.
+        const _path = config.api._path || config.api.path || '/otp/routers/default'
```

Comment:
Rename to `originalPath`

Reply:
Or `oldPath`. Same for `config.api._path`.